Gems/PhysX/Code/Source/EditorColliderComponent.cpp

```diff
@@ -379,9 +330,28 @@ namespace PhysX
         }
     }
 
+    bool EditorProxyShapeConfig::IsNonUniformlyScaledPrimitive() const
+    {
+        return m_hasNonUniformScale && (Utils::IsPrimitiveShape(GetCurrent()) || IsCylinderConfig());
+    }
+
     bool EditorProxyShapeConfig::ShowingSubdivisionLevel() const
     {
-        return (m_hasNonUniformScale && (IsCapsuleConfig() || IsSphereConfig() || IsAssetConfig()));
+        return m_hasNonUniformScale && (IsCapsuleConfig() || IsSphereConfig());
+    }
+
+    void EditorColliderComponent::Init()
+    {
+        // O3DE_DEPRECATION_NOTICE(GHI-14718)
+        // If initial value is PhysicsAsset, default to Box and update the UI.
+        // To be removed when m_shapeType intial value is changed to Box with GHI-14718.
+        if (m_shapeConfiguration.m_shapeType == Physics::ShapeType::PhysicsAsset)
+        {
+            m_shapeConfiguration.m_shapeType = Physics::ShapeType::Box;
+            AzToolsFramework::PropertyEditorGUIMessages::Bus::Broadcast(
+                &AzToolsFramework::PropertyEditorGUIMessages::RequestRefresh,
+                AzToolsFramework::PropertyModificationRefreshLevel::Refresh_AttributesAndValues);
+        }
```

Comment:
This entire cpp file is the same as before but removing anything to do with PhysX Asset (which will appear in the `EditorMeshColliderComponent.cpp` new file later).

This section is the part more worth highlighting.